Sources/OpenCombine/Publishers/OperatorSubscription.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  OperatorSubscription.swift
+//  
+//
+//  Created by Sergej Jaskiewicz on 26.06.2019.
+//
+
+internal class OperatorSubscription<Downstream: Subscriber>: CustomReflectable {
```

Comment:
Just interested - is explicit `internal` required by swiftlint?

Reply:
Yep

https://github.com/broadwaylamb/OpenCombine/blob/8b7bb88d66c4570cb37070d19d8f1f1f8735f8f5/.swiftlint.yml#L37